src/App.jsx

```diff
@@ -1,10 +1,22 @@
 import { ChakraProvider, Button, Image } from '@chakra-ui/react';
 import React, { useState } from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
 import './App.css';
 import DropZone from './components/DropZone/DropZone';
 import uploadImage from './util/furnitureUtils';
 import DonationForm from './components/DonationForm/DonationForm';
 
+import Dashboard from './pages/dashboard/Dashboard';
+import DonateEdit from './pages/dashboard/DonateEdit';
+import Drivers from './pages/dashboard/Drivers';
+import RoutesPage from './pages/dashboard/Routes';
+import Donate from './pages/donation/Donate';
```

Comment:
🚫 **[eslint]** <[import/extensions](https://github.com/import-js/eslint-plugin-import/blob/v2.26.0/docs/rules/extensions.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing file extension for "./pages/donation/Donate"